<launch> 
	<node pkg="joint_waypoint_controller" name="l_arm_joint_waypoint_controller" type="joint_waypoint_controller" output="screen" clear_params="true">
		<param name="filters" textfile="$(find joint_waypoint_controller)/launch/filters.xml"/>
		<param name="trajectory_type" value="cubic"/>
		<param name="spline_controller_prefix" value="/l_arm_trajectory_controller/"/>
	</node>
</launch>
